### PR TITLE
Fixed a bug that allowed csaber to parry when incapacitated

### PR DESCRIPTION
**Pull request by PrimeNumb**
## About the PR

This fixes a bug that allowed a csaber to parry another csaber even if the user was incapacitated.
For example: a person using treads and a stapled csaber arm would parry other csabers endlessly, despite being incapacitated.

## Why's this needed?

Removes (what I assume is) unintended behavior.

## Changelog

RichardGere:
+ Fixed a bug that allowed a csaber to parry even when the user was incapacitated

### CODE REVIEW

**Comment by ZeWaka** [comment]
We really need to make those statements into a proc or something, I think we have one but not sure.

File: code/obj/item/misc_weapons.dm [hunk 1]
```diff
@@ -99,7 +99,7 @@
 		var/obj/item/sword/S = H.find_type_in_hand(/obj/item/sword, "right")
 		if (!S)
 			S = H.find_type_in_hand(/obj/item/sword, "left")
-		if (S && S.active)
+		if (S && S.active && !(H.lying || H.hasStatus("stunned") || H.hasStatus("weakened") || H.hasStatus("paralysis")))
```
Comment (flrsh) [inline code/obj/item/misc_weapons.dm:102]:
```suggestion
		if (S && S.active && !(isalive(H) || H.lying || H.hasStatus("stunned") || H.hasStatus("weakened") || H.hasStatus("paralysis")))
```

Edge case: buckled in dead person.

Comment (PrimeNumb) [reply code/obj/item/misc_weapons.dm:102]: Good catch

**Comment by Kyle2143** [comment]
Believe it or not, this was intended. Or rather, I never cared to fix it because I thought it was funny. The same thing effects katanas.

**Pull request merged**